tests/integration_tests/functional/test_api.py

```diff
@@ -100,16 +100,27 @@ def test_api_put_update_pre_boot(test_microvm_with_api):
         'vcpu_count': 4,
         'ht_enabled': True,
         'mem_size_mib': 256,
-        'cpu_template': 'C3',
         'track_dirty_pages': True
     }
-    response = test_microvm.machine_cfg.put(
-        vcpu_count=microvm_config_json['vcpu_count'],
-        ht_enabled=microvm_config_json['ht_enabled'],
-        mem_size_mib=microvm_config_json['mem_size_mib'],
-        cpu_template=microvm_config_json['cpu_template'],
-        track_dirty_pages=microvm_config_json['track_dirty_pages']
-    )
+    if platform.machine() != 'aarch64':
```

Comment:
Nit: I would rather check if `platform.machine() == 'x86_64'` here. Since we only support templates for x86_64